### PR TITLE
CSS update: Stage1

### PR DESCRIPTION
**Pull request by mischakolbe**
Overall CSS changes. The app visually remained mostly the same, but it now uses the same styling for all Sender/Driver flow screens; Buttons are now always in the same place, etc.

This is the first of many styling updates and there are still tons of issues. I didn't go any further to get your guys' input early on, whether these changes are what everyone had in mind, when we talked about my involvement.

### CODE REVIEW

**Review by mAkeddar — COMMENTED** [review]
Nice one! 
I think the changes are going in the right direction :D
There are two behaviors that bother me a bit: 
- The scrolling "behind" the next and back buttons. I think that we agreed on buttons that appear at the end of scrolling (My fix was to include them as form.item, but that causes some issues in navigator mode ) (@hugo2410 @conradbielitz )
- The scrolling in the profile screen is not working, I think an overflowY is missing in the card div

**Comment by hugo2410** [comment]
@mischakolbe i Agree with @mAkeddar 's comment on the buttons' only appearing when we scroll right to the bottom. I have spent quite some time and Mehdi too. So if we realize it's not easy to do, maybe we can just go with this

**Comment by mischakolbe** [comment]
Thank you so much for the feedback, @mAkeddar & @hugo2410 ! Ok, buttons at the bottom, pushed down & scrolling if there is too much content.

I think I'll manage to do that with a general `div` for all our content, etc. Fingers crossed ^^

**Comment by mischakolbe** [comment]
The latest state should address the notes you had, @mAkeddar, besides various other things.

The landing page looks absolutely horrible - it's just a WIP, where I tried out some things.

**Comment by hugo2410** [comment]
We have our new css king :100:

**Comment by hugo2410** [comment]
THe progress bar on the route creation goes from 0 to 66% and is stuck on 66% for two screens

**Comment by hugo2410** [comment]
I find the full blue profile screen a bit overwhelming personally. But can be discussed

File: src/Pages/Driver/enterDeliveryCapacity.tsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ import { CAPACITY_OPTIONS } from '../../constant';
 
 const { Title } = Typography;
 const NEXT_SCREEN = '/deliver/routeSummary';
-const PROGRESS = 75;
+const PROGRESS = 66;
```
Comment (hugo2410) [inline src/Pages/Driver/enterDeliveryCapacity.tsx:14]: I'm guessing the progress needs to be adapted here

File: src/Pages/Driver/enterDrivingTime.tsx [hunk 2]
```diff
@@ -10,46 +10,36 @@ import SingleTripContent from '../../Components/PageComponents/SingleTripContent
 import RecurrentTripContent from '../../Components/PageComponents/RecurrentTripContent';
 
 const { Title } = Typography;
-const PROGRESS = 50;
+const PROGRESS = 66;
```
Comment (hugo2410) [inline src/Pages/Driver/enterDrivingTime.tsx:13]: same here

**Comment by mischakolbe** [comment]
Well spotted @hugo2410 ! Thank you!! I adjusted the percentage on the first screen to 33%.

FYI: I adjusted the driver flow to match the sender flow; The sender flow has 4 screens before the summary (0%, 25%, 50%, 75%), whereas the driver flow currently has 3 screens. It was 25%, 50%, 75% before, but to keep the same 0-100% progression I changed it to 0%, 33%, 66%, 100% (summary). And then I messed up and changed both middle screens to 66% xDDD Sorry!

**Comment by mischakolbe** [comment]
Regarding blue profile screen: Yep, I agree. It's pretty intense. I kinda like it, since it makes that screen super obviously different, but it's a bit much as-is...

**Review by hugo2410 — APPROVED** [review]

**Pull request merged**